src/benchmark_environments/classic.py

```diff
@@ -0,0 +1,32 @@
+"""Adaptation of classic Gym environments for IRL."""
```

Comment:
I think I would prefer `classic_control`, in so far as we intend this to form part of the "fixed Gym" subset of tasks it's good to mirror their API. If we do keep it `classic`, then we'd probably also need to put in other "classic" environments like cliffwalk here.

Reply:
Also nit: normally I see a newline after module docstring. (Although this does not actually seem to be in PEP8 or PEP257.)